CONTRIBUTING.md

```
@@ -0,0 +1,110 @@
+
+To run the tests, start by building the project as described in the [Setup Development Environment](#setup-development-environment) section.
+
+Then you can rerun the tests with:
+
+```bash
+python3 -m pytest testing
+```
+
+## Build Wheels
+
+_TBA_
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)